### PR TITLE
File access: tweak procedure on installing terminal and ssh add-on

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

File access: tweak procedure on installing terminal and ssh add-on
- rephrase it to use numbered steps for actionable steps
- move conceptional information to introduction

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/common-tasks/file_access.md [hunk 1]
```diff
@@ -35,13 +35,30 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
+If you want to use the Home Assistant command line or an SSH client, you can do this through the **Terminal & SSH** add-on.
 
-To use the add-on, enter a password or public key on its configuration page, then save and start the add-on.
+The **Terminal & SSH** add-on provides the following functionalities:
 
-The Terminal & SSH add-on also provides a web terminal which allows you to access a terminal via the Home Assistant user interface. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
+- It provides a web terminal that you can access from the Home Assistant user interface.
+- It allows you to use the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring backups, and more.
+  - For a list of command line commands, refer to [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/).
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
  - For a list of command line commands, refer to [Home Assistant via Command Line](/common-tasks/os#home-assistant-via-the-command-line).
```

File: source/_includes/common-tasks/file_access.md [hunk 2]
```diff
@@ -35,13 +35,30 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
+- It also includes nano and vi editors.
+
+Note: The Terminal & SSH add-on does *not* provide access to the underlying host file system.
+
+To get started with the **Terminal & SSH** add-on, follow these steps:
+
+1. In the bottom left, select your user to open the {% my profile title="**Profile**" %} page. Make sure **Advanced Mode** is activated.
+2. To install the add-on, go to the add-on store under [**Settings** > **Add-ons**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_ssh) and install the **Terminal & SSH** add-on.
+3. To use the web terminal, **start** the add-on, then select **Open Web UI**.
+   - You can now start typing your [commands](https://www.home-assistant.io/hassio/commandline/).
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
   - You can now start typing your [commands](/common-tasks/os#home-assistant-via-the-command-line).
```

File: source/_includes/common-tasks/file_access.md [hunk 3]
```diff
@@ -35,13 +35,30 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
+The **Terminal & SSH** add-on provides the following functionalities:
 
-The Terminal & SSH add-on also provides a web terminal which allows you to access a terminal via the Home Assistant user interface. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
+- It provides a web terminal that you can access from the Home Assistant user interface.
+- It allows you to use the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring backups, and more.
+  - For a list of command line commands, refer to [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/).
+- It allows connecting to your system using an SSH client.
+- It also includes nano and vi editors.
+
+Note: The Terminal & SSH add-on does *not* provide access to the underlying host file system.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
<div class="note">

    The Terminal & SSH add-on does *not* provide access to the underlying host file system.

</div>
```
Maybe wrap this in a note div?

File: source/_includes/common-tasks/file_access.md [hunk 4]
```diff
@@ -35,13 +35,30 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
+
+To get started with the **Terminal & SSH** add-on, follow these steps:
+
+1. In the bottom left, select your user to open the {% my profile title="**Profile**" %} page. Make sure **Advanced Mode** is activated.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
1. In the bottom left, select your user to open the {% my profile title="**Profile**" %} page. Make sure **Advanced Mode** is enabled.
```
Looks like we are mostly using "enabled" for this.

File: source/_includes/common-tasks/file_access.md [hunk 5]
```diff
@@ -35,13 +35,30 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
 
-To use the add-on, enter a password or public key on its configuration page, then save and start the add-on.
+The **Terminal & SSH** add-on provides the following functionalities:
 
-The Terminal & SSH add-on also provides a web terminal which allows you to access a terminal via the Home Assistant user interface. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
+- It provides a web terminal that you can access from the Home Assistant user interface.
+- It allows you to use the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring backups, and more.
+  - For a list of command line commands, refer to [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/).
+- It allows connecting to your system using an SSH client.
+- It also includes nano and vi editors.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
- It also includes common tools like nano and vi editors.
```
🤷‍♂️

File: source/_includes/common-tasks/file_access.md [hunk 6]
```diff
@@ -35,13 +35,30 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
+
+To get started with the **Terminal & SSH** add-on, follow these steps:
+
+1. In the bottom left, select your user to open the {% my profile title="**Profile**" %} page. Make sure **Advanced Mode** is activated.
+2. To install the add-on, go to the add-on store under [**Settings** > **Add-ons**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_ssh) and install the **Terminal & SSH** add-on.
+3. To use the web terminal, **start** the add-on, then select **Open Web UI**.
+   - You can now start typing your [commands](https://www.home-assistant.io/hassio/commandline/).
+4. If you want to access from an ssh client, you need to enter credentials:
+   - Open the **Configuration** page.
+   - Enter a password or public key.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
   - Enter a password or authorized Keys.
```
ref: https://github.com/home-assistant/addons/blob/master/ssh/translations/en.yaml#L4

File: source/_includes/common-tasks/file_access.md (hunk 3 above)
Comment (c0ffeeca7) [reply source/_includes/common-tasks/file_access.md]: Integrated it into the list instead.

File: source/_includes/common-tasks/file_access.md [hunk 7]
```diff
@@ -35,13 +35,29 @@ You would then enter the IP address or hostname of your Home Assistant OS instan
...
+The **Terminal & SSH** add-on provides the following functionalities:
 
-The Terminal & SSH add-on also provides a web terminal which allows you to access a terminal via the Home Assistant user interface. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
+- It provides a web terminal that you can access from the Home Assistant user interface.
+- It allows you to use the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring backups, and more.
+  - For a list of command line commands, refer to [Home Assistant via Command Line](/common-tasks/os#home-assistant-via-the-command-line).
+- It allows connecting to your system using an SSH client.
+- It also includes common tools like nano and vi editors.
+- The Terminal & SSH add-on does **not provide** access to the underlying host file system. For access to the host file system, refer to one of the other methods listed in this section, such as [using the Samba add-on](#installing-and-using-the-samba-add-on).
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
- The Terminal & SSH add-on does **not provide** access to the underlying host file system.
```
For that you would need to setup [debugging](https://developers.home-assistant.io/docs/operating-system/debugging) which is probably not something we would want to add here.

**Review by ludeeus — APPROVED** [review]
